src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_APPOINTMENTS_OVERVIEW;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.DateMatchesPredicate;
+
+/**
+ * Lists all Appointments in the appointment book to the user.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views all appointments [on a date]"
+            + "Parameters: "
+            + "[" + PREFIX_DATE + "DATE "
+            + "Example: " + COMMAND_WORD
+            + PREFIX_DATE + "2020-09-11 "
+            + "or \n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Viewed all Appointments";
+
+    private final DateMatchesPredicate predicate;
+    private final boolean isAll;
+
+    /**
+     * Creates a ViewCommand to list the appointments on a date.
+     */
+    public ViewCommand() {
+        this.predicate = null;
+        this.isAll = true;
+    }
+
```

Comment:
The boolean `isAll` seems to be unused. 
Maybe instead of using a boolean can consider using `PREDICATE_SHOW_ALL_APPOINTMENTS` in Model.java then there is no change in the implementation

Lemme know what you think about this! 😄

Reply:
Yup, I did use `PREDICATE_SHOW_ALL_APPOINTMENT`, but just due to bugs of ystd, I doubted it was related to it so I changed. After solving the bug, forgot to add back.
Thanks for the suggestion!